packages/ui/src/theme/Color.ts

```diff
@@ -25,6 +25,12 @@ export enum Color {
   Black50 = 'rgba(0, 0, 0, 0.5)',
   Black20 = 'rgba(0, 0, 0, 0.2)',
 
+  //Colors with 30% opacity
```

Comment:
@timurhamitov added color Dark100, Blue300, Green300 and Red300 with opacity 30%